### PR TITLE
Menu

### PR DESCRIPTION
**Pull request by ksheremet**

### CODE REVIEW

**Comment by ksheremet** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/dasfoo/rover-android-client/pull-request/36/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dasfoo/rover-android-client)

File: app/src/main/java/ch/sheremet/dasfoo/rover/android/client/video/VideoFragment.java [hunk 1]
```diff
@@ -97,16 +112,20 @@ public void onSurfaceTextureUpdated(final SurfaceTexture surface) {
...
-                    PropertyReader propertyReader = new PropertyReader(getActivity());
-                    Properties properties = propertyReader.getProperties("network.properties");
-                    String host = ((MainActivity) getActivity()).getHost();
-                    int port = ((MainActivity) getActivity()).getPort();
-                    String password = properties.getProperty("password");
+                    // TODO(ksheremet): check errors
+                    final SharedPreferences sharedPreferences =
+                            PreferenceManager.getDefaultSharedPreferences(getActivity());
+                    final String host = sharedPreferences
+                            .getString(Settings.VIDEO_HOST.toString(), "Get host");
```
Comment (dotdoom) [inline app/src/main/java/ch/sheremet/dasfoo/rover/android/client/video/VideoFragment.java]: What is the second parameter to `getString()`?

Comment (ksheremet) [inline app/src/main/java/ch/sheremet/dasfoo/rover/android/client/video/VideoFragment.java]: Value to return if this preference does not exist. Maybe is it better to leave it empty?

**Comment by dotdoom** [comment]
:shipit:

[Approved with PullApprove](https://pullapprove.com/dasfoo/rover-android-client/pull-request/36/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dasfoo/rover-android-client)

**Comment by ksheremet** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/dasfoo/rover-android-client/pull-request/36/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=dasfoo/rover-android-client)

**Pull request merged**